### PR TITLE
Add birch, acacia, spruce, jungle, dark oak feature parity

### PR DESCRIPTION
**Pull request by HSQuillin**
Description
---
This pull request adds textures in `blocks.png` defines the new wood types in `Block.java`, allowing more feature parity and more variants in wood types. I only added certain wood types because some of the other, newer ones are redundant and do not fit in without overhauls to the project's aesthetic, or do not make sense in the context of the overworld.

Sample Image
---

image

Features Added
---

**These are the base wood types:**
- Birch
  - Leaves (6, 0)
  - Logs (5, 1, 6, 1)
  - Planks (5, 0)
- Acacia
  - Leaves (10, 0)
  - Logs (9, 1, 10, 1)
  - Planks (9, 0)
- Spruce
  - Leaves (8, 0)
  - Logs (7, 1, 8, 1)
  - Planks (7, 0)
- Jungle
  - Leaves (12, 0)
  - Logs (11, 1, 12, 1)
  - Planks (11, 0)
- Dark Oak
  - Leaves (14, 0)
  - Logs (13, 1, 14, 1)
  - Planks (13, 0)

Summary
---
In short, this adds fifteen new block types, with three variants of leaves, logs, and planks per each new type of wood. These do not naturally generate, but are to increase the palette of blocks to choose from. All textures created by me.

### CODE REVIEW

**Comment by BlueStaggo** [comment]
Hmm, they look nice. I won't merge this though, since I want to use different wood types and I want to store them differently. I want to flatten Block IDs, so the way you registered them might work out. Good textures though, I like them.
